packages/components/src/stories/DatasetDesignerPanels.tsx

```diff
@@ -18,10 +18,11 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { withKnobs } from '@storybook/addon-knobs';
 
-import { NEW_DATASET_MODEL } from '../test/data/constants';
+import {NEW_DATASET_MODEL_WITH_DATASPACE, NEW_DATASET_MODEL_WITHOUT_DATASPACE} from '../test/data/constants';
 import { DatasetDesignerPanels } from '../components/domainproperties/dataset/DatasetDesignerPanels';
 import { DatasetModel } from '../components/domainproperties/dataset/models';
 import getDatasetDesign from '../test/data/dataset-getDatasetDesign.json';
+import getDatasetDesignSharedStudy from '../test/data/dataset-getDatasetDesignSharedStudy.json';
 
 class NewDatasetDesigner extends React.PureComponent<any, any> {
```

Comment:
I think we might need to re-think these stories. Do we need to have different stories based on different study types (i.e. dataspace vs visit based vs date based). Also, if I pull up storybook and go to the edit ones currently it seems like things are disabled (i.e. the case for the "dataset created from assay source").

Reply:
we have dataspace and non dataspace Visit based stories. I am not sure how to mock date based without changing the getServerContext().moduleContext.study.timepointType variable. Fixed the disabled fields.